nms/app/packages/magmalte/app/views/subscriber/SubscriberOverview.js

```diff
@@ -254,223 +213,46 @@ function SubscriberInternal(props: WithAlert) {
-                          });
-
-                          return (
-                            <ActionTable
-                              data-testid="detailPanel"
-                              title=""
-                              data={subscriberSessionRows}
-                              columns={[
-                                {title: 'APN Name', field: 'apnName'},
-                                {title: 'Session ID', field: 'sessionId'},
-                                {title: 'State', field: 'state'},
-                                {title: 'IP Address', field: 'ipAddr'},
-                                {
-                                  title: 'Active Duration',
-                                  field: 'activeDuration',
-                                },
-                                {
-                                  title: 'Active Policy IDs',
-                                  field: 'activePolicies',
-                                  render: currRow =>
-                                    currRow.activePolicies.length ? (
-                                      <List>
-                                        {currRow.activePolicies.map(policy => (
-                                          <ListItem key={policy.id}>
-                                            <Link>{policy.id} </Link>
-                                          </ListItem>
-                                        ))}
-                                      </List>
-                                    ) : (
-                                      <Text>{'-'}</Text>
-                                    ),
-                                },
-                              ]}
-                              options={{
-                                actionsColumnIndex: -1,
-                                pageSizeOptions: [5],
-                                toolbar: false,
-                                paging: false,
-                                rowStyle: {background: '#f7f7f7'},
-                                headerStyle: {
-                                  background: '#f7f7f7',
-                                  color: colors.primary.comet,
-                                },
-                              }}
-                            />
-                          );
-                        },
-                      },
-                    ]
-              }
-            />
-          </div>
-        ) : (
-          '<Text>No Subscribers Found</Text>'
-        )}
+        <SubscriberTabs
+          value={currTabIndex}
+          onChange={(_, newIndex: number) => setCurrTabIndex(newIndex)}
+          variant="fullWidth">
+          {subscribersTabList.map((k: string, idx: number) => {
+            return (
+              <SubscriberTab key={idx} label={k} className={classes.tab} />
+            );
+          })}
+        </SubscriberTabs>
+        {currTabIndex === 0 && <Subscriber columns={tableColumns} />}
```

Comment:
Can you add check here to display subscriber table only for LTE networks and not for FEG_LTE networks